src/middleware/updateValidation.js

```diff
@@ -0,0 +1,46 @@
+const constants = require('../lib/constants');
+
+// eslint-disable-next-line consistent-return
+const validateUpdate = (req, res, next) => {
+  const { password, passwordConfirmation, address, age, phoneNumber } =
+    req.body;
+  const errorsArray = [];
+  const properties = Object.keys(req.body);
+  // eslint-disable-next-line consistent-return
+  properties.forEach((property) => {
+    if (req.body[property] === '') {
+      errorsArray.push(`${property} Cannot be empty`);
+    }
+  });
+  if (password) {
```

Comment:
You might also want to check the password if it satisfies our password code:
```
PASSWORD_REGEX: new RegExp(/^(?=.*?[A-Z])(?=(.*[a-z]){1,})(?=(.*[\d]){1,})(?=(.*[\W]){1,})(?!.*\s).{7,}$/)
```

Reply:
yes , it's checked in the update validation middleware, this is just checking if it's provided